packages/flutter/lib/src/widgets/bottom_navigation_bar_item.dart

```diff
@@ -96,4 +97,7 @@ class BottomNavigationBarItem {
   ///  * [Icon.color] and [ImageIcon.color] to control the foreground color of
   ///    the icons themselves.
   final Color? backgroundColor;
+
+  /// The text to display in the tooltip for this [BottomNavigationBarItem].
+  final String? toolTip;
```

Comment:
The `BottomNavigationBarItem` class is used by both the Material bottom navigation bar and the Cupertino bottom navigation bar (which doesn't show a tooltip when the item is long-pressed). Could you add that to the documentation? The documentation for `label` may also need updating.

Reply:
The "disable tooltip" behavior doesn't seem to be in the documentation.